x/ccv/provider/keeper/proposal.go

```diff
@@ -299,17 +299,17 @@ func (k Keeper) MakeConsumerGenesis(ctx sdk.Context, prop *types.ConsumerAdditio
 // SetPendingConsumerAdditionProp stores a pending consumer addition proposal.
 //
 // Note that the pending consumer addition proposals are stored under keys with
-// the following format: PendingCAPBytePrefix | len(spawnTime) | spawnTime | chainID
+// the following format: PendingCAPBytePrefix | spawnTime | chainID
 // Thus, if multiple consumer addition proposal for the same chain will pass at
 // the same time, then only the last one will be stored.
-func (k Keeper) SetPendingConsumerAdditionProp(ctx sdk.Context, clientInfo *types.ConsumerAdditionProposal) {
+func (k Keeper) SetPendingConsumerAdditionProp(ctx sdk.Context, prop *types.ConsumerAdditionProposal) {
 	store := ctx.KVStore(k.storeKey)
-	bz, err := k.cdc.Marshal(clientInfo)
+	bz, err := k.cdc.Marshal(prop)
```

Comment:
I'm not sure why we used `k.cdc.Marshal` and not `prop.Marshal()`?

Is there a significant difference?